### PR TITLE
Fix custom submanager names

### PR DESCRIPTION
**Pull request by matteovg7**

### CODE REVIEW

File: src/MShop.php [hunk 1]
```diff
@@ -73,8 +73,12 @@ public static function create( \Aimeos\MShop\ContextIface $context,
 		if( empty( $name ) )
 		{
 			$subpath = !empty( $parts ) ? join( '/', $parts ) . '/' : '';
-			$name = $config->get( 'mshop/' . $domain . '/manager/' . $subpath . 'name' );
-			$name = $name ?: $config->get( 'mshop/' . $domain . '/manager/name', 'Standard' );
```
Comment (aimeos) [inline src/MShop.php:77]: If a name is passed as argument to that method, it's not used any more in your code

**Comment by aimeos** [comment]
What's the problem with the current implementations? Can you give an example when it will fail?

**Comment by aimeos** [comment]
We've shortend and improved the code a bit so it passes the tests :-)

**Comment by aimeos** [comment]
The patch has been backported to the 2022.07 branch and a new release (2022.07.6) has been tagged. Can you please confirm that this fixes the problem?

**Comment by matteovg7** [comment]
Ok, now it's all ok. Thank you!

**Comment by matteovg7** [comment]
Sorry, I reopen this merge request because I noticed that from release 2022.07.7 the code was reverted and now I had the same problem that I had before.
Why the code was modify from version 2022.07.6? If I need to change the structure of config file to a new format please tell me how.

**Comment by aimeos** [comment]
The change crashed other code parts and would have led to a heavy breaking change in the stable release. In the current dev-master branch, this has been changed again to simplify configuration but until (and including 2022.10 LTS), you should be able to configure your managers in your `./config/mshop.php` using:

```php
return [
    'customer' => [
        'manager' => [
            'name' => 'Myproject',
            'address' => [
                'name' => 'Standard'
            ],
            'list' => [
                'name' => 'Myproject',
                'type' => [
                    'name' => 'Standard'
                ]
            ],
            'property' => [
                'name' => 'Myproject',
                'type' => [
                    'name' => 'Standard'
                ]
            ]
        ]
    ]
];
```

**Comment by matteovg7** [comment]
No, it doesn't work for me.
This is my config file:

```
return [
    'coupon' => [
        'manager' => [
            'name' => 'CouponVG7',
            'code' => [
                'name' => 'CouponCodeVG7',
                'decorators' => [
                    'local' => ['CouponCodeCustomerId']
                ],
            ]
        ]
    ],
    'customer' => [
        'manager' => [
            'name' => 'CustomerVG7',
            'decorators' => [
                'local' => ['UsersIdPrint']
            ],
            'address' => [
                'name' => 'AddressVG7'
            ]
        ]
    ],
    'locale' => [
        'manager' => [
            'name' => 'LocaleVG7',
        ]
    ],
    'media' => [
        'manager' => [
            'name' => 'MediaVG7',
            'decorators' => [
                'local' => ['MediaIdPrint']
            ],
        ]
    ],
    'order' => [
        'manager' => [
            'name' => 'OrderVG7',
            'decorators' => [
                'local' => ['OrderIdPrint']
            ],
            'base' => [
                'name' => 'OrderBaseVG7',
                'address' => [
                    'name' => 'OrderBaseAddressVG7',
                    'decorators' => [
                        'local' => ['OrderBaseAddressAddFields']
                    ],
                ],
                'product' => [
                    'name' => 'OrderBaseProductVG7',
                    'decorators' => [
                        'local' => ['OrderBaseProductIdPrint']
                    ],
                ],
                'service' => [
                    'name' => 'OrderBaseServiceVG7'
                ],
            ]
        ]
    ],
    'product' => [
        'manager' => [
            'name' => 'ProductVG7',
            'decorators' => [
                'local' => ['ProductIdPrint', 'ProductCodePrint']
            ],
        ]
    ],
    'service' => [
        'manager' => [
            'name' => 'ServiceVG7',
            'decorators' => [
                'local' => ['ServiceIdPrint']
            ],
        ]
    ],
];
```

and I have this exception for example:
`Class "\Aimeos\MShop\Locale\Manager\Site\LocaleVG7" not available`
but I don't have extended the Aimeos\MShop\Locale\Manager\Site\Standard class in my extension.

I also tried to edit the locale array like this, but nothing change:
```
    'locale' => [
        'manager' => [
            'name' => 'LocaleVG7',
        ],
        'site' => [
            'name' => 'Standard',
        ]
    ],
```

**Comment by aimeos** [comment]
Your configuration must be:
```
    'locale' => [
        'manager' => [
            'name' => 'LocaleVG7',
            'currency' => [
                'name' => 'Standard',
            ],
            'language' => [
                'name' => 'Standard',
            ],
            'site' => [
                'name' => 'Standard',
            ]
        ],
    ],
```

**Pull request merged**